docs/sources/developers/plugins/package-a-plugin.md

```diff
@@ -0,0 +1,65 @@
+---
+description: How to package a plugin
+title: Package a plugin
+---
+
+You've just built your first plugin and you're getting ready to [publish it](publish-a-plugin.md), and now you need to package it. This step follows the building and [signing](sign-a-plugin.md) steps for plugin deployment. 
```

Comment:
You might not want to publish, you might just be looking to package and use locally.

Reply:
ok